stdlib/2/pydoc.pyi

```diff
@@ -0,0 +1,176 @@
+def visiblename(name: str, all: Optional[List[str]] = ..., obj: Optional[object] = ...) -> bool: ...
+def classify_class_attrs(object: object) -> List[Tuple[str, str, type, str]]: ...
+
+def ispackage(path: str) -> bool: ...
+def source_synopsis(file: IO[AnyStr]) -> Optional[str]: ...
+def synopsis(filename: str, cache: MutableMapping[str, Tuple[int, str]] = ...) -> str: ...
+
+class ErrorDuringImport(Exception):
+    filename = ...  # type: str
+    exc = ...  # type: Optional[Type[BaseException]]
+    value = ...  # type: Optional[BaseException]
+    tb = ...  # type: Optional[TracebackType]
+    def __init__(self, filename: str, exc_info: _Exc_Info) -> None: ...
+
+def importfile(path: str) -> ModuleType: ...
+def safeimport(path: str, forceload: bool = ..., cache: MutableMapping[str, ModuleType] = ...) -> ModuleType: ...
+
+class Doc:
+    def document(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def fail(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def docmodule(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def docclass(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def docroutine(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def docother(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def docproperty(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def docdata(self, object: object, name: Optional[str] = ..., *args: Any) -> str: ...
+    def getdocloc(self, object: object) -> Optional[str]: ...
+
+class HTMLRepr(Repr):
+    maxlist = ...  # type: int
+    maxdict = ...  # type: int
+    maxstring = ...  # type: int
+    def __init__(self) -> None: ...
+    def escape(self, text: str) -> str: ...
+    def repr(self, object: object) -> str: ...
+    def repr1(self, x: object, level: complex) -> str: ...
+    def repr_string(self, x: AnyStr, level: complex) -> str: ...
+    def repr_str(self, x: AnyStr, level: complex) -> str: ...
+    def repr_instance(self, x: object, level: complex) -> str: ...
+    def repr_unicode(self, x: AnyStr, level: complex) -> str: ...
+
+class HTMLDoc(Doc):
+    def repr(self, object: object) -> str: ...
+    def escape(self, test: str) -> str: ...
+    def page(self, title: str, contents: str) -> str: ...
+    def heading(self, title: str, fgcol: str, bgcol: str, extras: str = ...) -> str: ...
+    def section(self, title: str, fgcol: str, bgcol: str, contents: str, width: int = ..., prelude: str = ..., marginalia: Optional[str] = ..., gap: str = ...) -> str: ...
+    def bigsection(self, title: str, *args) -> str: ...
+    def preformat(self, text: str) -> str: ...
+    def multicolumn(self, list: List[Any], format: Callable[[Any], str], cols: int = ...) -> str: ...
+    def grey(self, text: str) -> str: ...
+    def namelink(self, name: str, *dicts: Tuple[MutableMapping[str, str]]) -> str: ...
```

Comment:
Should remove the Tuple part. The annotation for an `*args` type is the type of an individual arg.